### PR TITLE
fix: form view UI fixes

### LINKED ISSUE #6502: 🐛 Bug: Form view UI/UX corrections
Additional UI/UX corrections related to form view
- [x] Title & description should have line break
- [x] Title and description - expand text area as text grows
- [x] Form label text - overflows form area
  - [x] wrap & show complete text. ensure line break is taken care. in both builder & shared form 
- [x] Remove success toast message for form
- [x] Long text
  - [x] Top padding
  - [x] Resize didn't work 
  - [x] Resize. Fill data. Click outside - Long text shrinks as single line text. Should show at least few lines worth of data
Screenshot 2023-09-27 at 4 02 58 PM
- [x] Checkbox, left aligned. Make entire line clickable (as in default value configuration)
  - [x] in builder, mark it at centre 
- [x] Attachment - attach icon should always persist
  - [x] show only image preview or tile after its attached in form
  - [x] do not allow a file with same name to be uploaded again
- [x] Text box - should not increase height on click
- [x] Active input box, mark with a thicker border

### PR DESCRIPTION
**Pull request by sreehari2003**
## Change Summary

Provide summary of changes with issue number if any.

- closes #6502

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [x] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

**Comment by sreehari2003** [comment]
image

**Comment by sreehari2003** [comment]
## text was breaking before

image

**Comment by sreehari2003** [comment]
## moved nc buttons 

image


image

File: packages/nc-gui/pages/index/[typeOrId]/form/[viewId]/index/index.vue [hunk 1]
```diff
@@ -72,14 +72,17 @@ const onDecode = async (scannedCodeValue: string) => {
 <template>
   <div class="h-full flex flex-col items-center">
     <div
-      class="color-transition relative flex flex-col justify-center gap-2 w-full max-w-[max(33%,600px)] m-auto py-4 pb-8 px-16 md:(bg-white dark:bg-slate-700 rounded-lg border-1 border-gray-200 shadow-xl)"
+      class="color-transition flex flex-col justify-center gap-2 w-full max-w-[max(33%,600px)] m-auto py-4 pb-8 px-16 md:(bg-white dark:bg-slate-700 rounded-lg border-1 border-gray-200 shadow-xl)"
     >
       <template v-if="sharedFormView">
-        <h1 class="prose-2xl font-bold self-center my-4" style="word-break: break-all">
+        <h1 class="prose-2xl font-bold self-center my-4 break-words">
```
Comment (wingkwong) [inline packages/nc-gui/pages/index/[typeOrId]/form/[viewId]/index/index.vue:78]: `break-words` gives `overflow-wrap: break-word;`.  It should be `break-all` for `word-break: break-all`.

Ignore this comment if you switch to use `break-word`.

File: packages/nc-gui/components/smartsheet/Form.vue [hunk 2]
```diff
@@ -422,27 +422,27 @@ watch(view, (nextView) => {
           </div>
 
           <div class="flex flex-wrap gap-2 mb-4">
-            <button
+            <NcButton
               v-if="hiddenColumns.length"
-              type="button"
-              class="nc-form-add-all color-transition bg-white transform hover:(text-primary ring ring-accent ring-opacity-100) active:translate-y-[1px] px-2 py-1 shadow-md rounded"
+              type="secondary"
```
Comment (wingkwong) [inline packages/nc-gui/components/smartsheet/Form.vue:427]: No such `secondary` type for NcButton.

```
'link' | 'default' | 'primary' | 'ghost' | 'dashed' | 'text' | 'danger' | undefined
```

Comment (sreehari2003) [reply packages/nc-gui/components/smartsheet/Form.vue:427]:
```css
.nc-button.ant-btn-secondary {
  @apply bg-white border-1 border-gray-200 text-gray-700 md:(hover:bg-gray-100);
}
``` 

there is  secondary class in ant and we are over riding  it

Comment (wingkwong) [reply packages/nc-gui/components/smartsheet/Form.vue:427]: From ant design webpage, it seems that `secondary` is `default`. I guess we have default, primary, and secondary. Anyway, adding `secondary` to Button props is fine to me.

**Comment by sreehari2003** [comment]
image

**Review by dstala — APPROVED** [review]
Pending for later
- warning if file with same name is uploaded (attachment field)
- single select field, edit mode remain in bold state (corner case)

**Pull request merged**